pkg/oc/admin/migrate/storage/storage.go

```diff
@@ -161,6 +164,9 @@ func NewCmdMigrateAPIStorage(name, fullName string, f *clientcmd.Factory, in io.
 		Long:    internalMigrateStorageLong,
 		Example: fmt.Sprintf(internalMigrateStorageExample, fullName),
 		Run: func(cmd *cobra.Command, args []string) {
+			apiextensioninstall.Install(legacyscheme.GroupFactoryRegistry, legacyscheme.Registry, legacyscheme.Scheme)
+			apiregistrationinstall.Install(legacyscheme.GroupFactoryRegistry, legacyscheme.Registry, legacyscheme.Scheme)
```

Comment:
need a follow up to figure out what we need to do to revert this... migrate shouldn't need registered types... it reads into unstructured and puts back

Reply:
> need a follow up to figure out what we need to do to revert this... migrate shouldn't need registered types... it reads into unstructured and puts back

done